float32 change_latitude
float32 change_longitude
float32 change_height
float32 gripper_rot
float32 duration
bool absolute_latitude
bool absolute_longitude
bool absolute_height
---
float32 cur_latitude
float32 cur_longitude
float32 cur_height
---
